examples/python-betterproto/test.sh

```
@@ -0,0 +1,31 @@
+
+# Initialize and generate
+bufrnix_init
+bufrnix
+
+echo ""
+echo "Generated files:"
+find proto/gen -name "*.py" -type f | sort
+
+echo ""
+echo "Betterproto verification:"
+if find proto/gen -name "*.py" -type f | xargs grep -l "@dataclass" > /dev/null 2>&1; then
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)